README.md

```diff
@@ -85,24 +85,24 @@ tfswitch #will automatically switch to terraform version 0.14.4
 3. Hit **Enter** to install.
 ### Install latest implicit version for stable releases
 1. Install the latest implicit stable version.
-2. Ex: `tfswitch -s 0.13` or `tfswitch --latest-stable 0.13` downloads 0.13.6 (latest) version.
+2. Ex: `tfswitch -s 0.13` or `tfswitch --latest-stable 0.13` downloads <1.1.0, >=1.0.0, 1.0 downloads <2.0.0, >=1.0.0.
```

Comment:
I had to re-read several times to make myself sure I get the things, hence the suggestion from regular user's standpoint:
```suggestion
2. Ex: `tfswitch -s 0.13` or `tfswitch --latest-stable 0.13` downloads `>=1.0.0, <1.1.0`, and `tfswitch -s 1.0` or `tfswitch --latest-stable 1.0` downloads `>=1.0.0, <2.0.0`.
```
Also would it worth to mention the logic is the same as what pessimistic constraint operator does in TF version constraint?
https://www.terraform.io/language/expressions/version-constraints#-3

Reply:
I will try to reword it